### PR TITLE
Bump khard and take over proxied maintenance

### PR DESCRIPTION
**Pull request by holgersson32644**
This is a replacement of PR #1578 -- taking over wouldn't make much sense after the huge amount of time.

### CODE REVIEW

File: app-misc/khard/khard-0.17.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+RDEPEND="
+	dev-python/atomicwrites[${PYTHON_USEDEP}]
+	dev-python/configobj[${PYTHON_USEDEP}]
+	dev-python/ruamel-yaml[${PYTHON_USEDEP}]
+	dev-python/ruamel-yaml-clib[${PYTHON_USEDEP}]
+	dev-python/unidecode[${PYTHON_USEDEP}]
+	dev-python/vobject[${PYTHON_USEDEP}]
+"
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
```
Comment (DarthGandalf) [inline app-misc/khard/khard-0.17.0.ebuild:34]: This is handled by DISTUTILS_USE_SETUPTOOLS, and why do you set it as rdepend if it's added to DEPEND here?

File: app-misc/khard/khard-0.17.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+src_install() {
+	distutils-r1_src_install
+
+	if use zsh-completion; then
+		insinto /usr/share/zsh/site-functions
+		doins misc/zsh/_khard
+	fi
+}
+
+python_test() {
```
Comment (DarthGandalf) [inline app-misc/khard/khard-0.17.0.ebuild:52]: `distutils_enable_tests` handles this and `test?` in DEPEND and more

File: app-misc/khard/khard-0.17.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+inherit distutils-r1
+
+DESCRIPTION="Console CardDAV client"
+HOMEPAGE="https://github.com/scheibler/khard"
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="test zsh-completion"
+RESTRICT="!test? ( test )"
+
+if [[ "${PV}" == *9999 ]]; then
```
Comment (DarthGandalf) [inline app-misc/khard/khard-0.17.0.ebuild:17]: Why do you need this if there is no 9999 ebuild?

File: app-misc/khard/khard-0.17.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+		dev-python/setuptools_scm[${PYTHON_USEDEP}]
+	)
+"
+
+DOCS=( CHANGES CONTRIBUTING.rst README.md doc/source/examples/khard.conf.example )
+
+src_install() {
+	distutils-r1_src_install
+
+	if use zsh-completion; then
```
Comment (DarthGandalf) [inline app-misc/khard/khard-0.17.0.ebuild:46]: No need

https://devmanual.gentoo.org/general-concepts/use-flags/index.html

**Review by andrey-utkin — APPROVED** [review]
Thanks a lot for this, @holgersson32644 .
For the record, I have checked your contribution history and your proxy-maint footprint on the current Gentoo repo to make sure you are not a random ghost (just due diligence, nothing personal). Seems you are very active and well recognized contributor, so I am not worried about that.

Thanks to @DarthGandalf for the review - it is useful to see which problems are spotted and even more useful to have an indication of which kinds of problems have _not_ been spotted. I think all the questions raised can be discounted for this version bump. These issues have already been there and are my fault. We should handle all of this later, separately.
I am not a fan of lumping all the existing small issues in one sweeping change. In fact I tend to stumble even when such small things like `DISTUTILS_USE_SETUPTOOLS=rdepend` go unexplained - it looks sensible now that i've done my research, but please do comment on what you're doing and why, it saves time to the reviewers.

The updated ebuilds installed correctly, I am pushing it to the central repo now.

**Comment by andrey-utkin** [comment]
Pushed as efa5df5a564ec07144292970ba0cafd0f43b78d1..712608c6c116e1395b0e6539a2620fe14ba41743

BTW @holgersson32644 , @DarthGandalf have you considered becoming Gentoo developer yourselves?

**Comment by DarthGandalf** [comment]
@andrey-utkin yes, I'm waiting for the developer who was going to mentor me to start doing that :)

**Comment by andrey-utkin** [comment]
@DarthGandalf I presume you are already working on [the quiz](https://projects.gentoo.org/comrel/recruiters/quizzes/ebuild-maintainer-quiz.txt)? Because you don't need to wait for anybody to start doing that, and it takes a substantial amount of time and effort to complete.

**Comment by DarthGandalf** [comment]
Yeah, I finished it already, though mentor didn't review yet

**Pull request closed without merging**